crates/client/rpc/src/trace_api.rs

```diff
@@ -99,11 +98,39 @@ where
             })?
             .map_err(|e| {
                 error!("Failed to call function: {:#?}", e);
-                StarknetRpcApiError::ContractError
+                StarknetRpcApiError::from(e)
             })?;
 
-        let simulated_transactions =
-            tx_execution_infos_to_simulated_transactions(tx_types, res).map_err(StarknetRpcApiError::from)?;
+        let mut simulated_transactions = vec![];
```

Comment:
why did we remove the functions like `tx_execution_infos_to_simulated_transactions` and `blockifier_to_starknet_rs_ordered_events` and replace the code directly?

Reply:
Maybe it was the only place it was used?

Reply:
yep those functions were only used once.